cmd/pullrequest-init/disk_test.go

```diff
@@ -111,7 +119,11 @@ func TestToDisk(t *testing.T) {
 	}
 	labels := map[string]struct{}{}
 	for _, fi := range fis {
-		labels[fi.Name()] = struct{}{}
+		text, err := url.QueryUnescape(fi.Name())
+		if err != nil {
+			t.Errorf("Error decoding label text: %s", fi.Name())
+		}
```

Comment:
i mean at this point i think it could make sense to have some unit tests directly for the functions you changed, what do you think? i think especially as we have to update this logic having some smaller more focused tests makes maintenance a bit easier

Reply:
Done.